src/Sylius/Bundle/ApiBundle/Tests/ApiPlatform/Bridge/Symfony/Routing/CachedRouteNameResolverTest.php

```diff
@@ -0,0 +1,321 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Tests\ApiPlatform\Bridge\Symfony\Routing;
+
+use ApiPlatform\Core\Api\OperationType;
+use ApiPlatform\Core\Bridge\Symfony\Routing\RouteNameResolverInterface;
+use ApiPlatform\Core\Exception\InvalidArgumentException;
+use PHPUnit\Framework\TestCase;
+use Prophecy\Argument;
+use Psr\Cache\CacheException;
+use Psr\Cache\CacheItemInterface;
+use Psr\Cache\CacheItemPoolInterface;
+use Sylius\Bundle\ApiBundle\ApiPlatform\Bridge\Symfony\Routing\CachedRouteNameResolver;
+use Sylius\Bundle\ApiBundle\Provider\RequestApiPathPrefixProviderInterface;
+
+final class CachedRouteNameResolverTest extends TestCase
+{
+    /**
+     * @test
+     */
+    public function testConstruct(): void
```

Comment:
```suggestion
    /**
     * @test
     */
    public function testConstruct(): void
```
PHPUnit requires to have either `@test` annotation or starting method name with `test`. As we already have annotation, let write it in a spec manner. This test also doesn't provide any value IMHO.